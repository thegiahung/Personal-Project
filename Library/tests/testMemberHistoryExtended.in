Member history 100000
Add member Python
Add collection books.csv
Member history 100000
Member history 100001
Rent 100000 111111
Rent 100000 111112
Rent 100000 111113
Member history 100000
Relinquish 100000 111111
Relinquish 100000 111112
Member history 100000
Relinquish 100000 111113
Member history 100000
Exit
