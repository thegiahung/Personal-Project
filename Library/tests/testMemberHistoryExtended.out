user: No members in system.

user: Success.

user: 17 books successfully added.

user: No rental history for member.

user: No such member in system.

user: Success.

user: Success.

user: Success.

user: No rental history for member.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: Ending Library process.
